lib/promscrape/config.go

```diff
@@ -102,8 +102,8 @@ type FileSDConfig struct {
 //
 // See https://prometheus.io/docs/prometheus/latest/configuration/configuration/#static_config
 type StaticConfig struct {
-	Targets []string          `yaml:"targets"`
-	Labels  map[string]string `yaml:"labels"`
+	Targets []string          `yaml:"targets,omitempty"`
```

Comment:
I'm unsure whether `StaticConfig` is usable without `targets` list.

Reply:
you're right
I fixed